### PR TITLE
BP-4512 Query param include_staff=false excludes staff from issuer de…

### PR DESCRIPTION
**Pull request by jasonKufner**
…tail response

### CODE REVIEW

File: apps/issuer/serializers_v2.py [hunk 1]
```diff
@@ -199,6 +199,12 @@ def update(self, instance, validated_data):
 
         return super(IssuerSerializerV2, self).update(instance, validated_data)
 
+    def to_representation(self, instance):
+        include_staff = self.context['request'].query_params.get('include_staff', 'true').lower()
```
Comment (clamothe) [inline apps/issuer/serializers_v2.py:203]: The `_scrub_boolean` method is used elsewhere in badgr-server for parsing booleans from a query param

File: apps/issuer/tests/test_issuer.py [hunk 2]
```diff
@@ -194,6 +195,48 @@ def test_update_issuer_does_not_clear_badgrDomain(self):
...
+        test_issuer = self.setup_issuer(owner=test_user, name='1')
+        issuer_data = {
+            'name': 'Testy MctestAlot',
+            'description': test_issuer.description,
+            'email': test_user.email,
+            'url': 'http://example.com'
+        }
+
+        staff_get_url = '/v1/issuer/issuers/{slug}/staff'.format(slug=test_issuer.entity_id)
+        url_and_query_param = set_url_query_params('/v2/issuers/{}'.format(test_issuer.entity_id), include_staff='false')
```
Comment (clamothe) [inline apps/issuer/tests/test_issuer.py:210]: It could be worth testing include_staff='true' as well

Comment (jasonKufner) [reply apps/issuer/tests/test_issuer.py:210]: done

File: apps/issuer/serializers_v2.py (hunk 1 above)
Comment (jasonKufner) [reply apps/issuer/serializers_v2.py]: done

**Review by ottonomy — APPROVED** [review]

**Pull request merged**